### DISCUSSION TITLE
Click-able Logo in Navbar

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#426] [Ideas] [answered]
I was wondering if theres a specific reason why Alkemio logo isn't a clickable. 

Making it redirect to home would feel very intuitive. 

![image](https://user-images.githubusercontent.com/64191883/133421950-6a36473b-f8d8-45cb-bd0d-d4c993f725fd.png)

**Comment by techsmyth** [comment, marked as answer]
Agreed, this would be a nice addition. Captured as https://github.com/alkem-io/client-web/issues/1115
